data/update/mysql/dbupdate-5.4.0-to-6.13.0.sql

```diff
@@ -2,6 +2,9 @@ SET default_storage_engine=InnoDB;
 -- Set storage engine schema version number
 UPDATE ezsite_data SET value='6.13.0' WHERE name='ezpublish-version';
 
+-- IMPORTANT:
+-- * Only covers database schema changes in eZ Platform, for other bundles you use (enterprise or otherwise), see doc!
```

Comment:
Doc link maybe? Or if there's no general doc for this and you're referring to doc for each bundle, then ok as is, or "see doc for each bundle".

Reply:
Better? _(if so I'll apply all over)_
```suggestion
-- * This only covers eZ Platform kernel, see doc for all steps (incl. database schema) needed when updating: https://doc.ezplatform.com/en/2.5/updating/updating_ez_platform/
```

Reply:
Looks good.